rest/admin_api.go

```diff
@@ -149,6 +155,13 @@ func (h *handler) handleCreateDB() error {
 	return base.HTTPErrorf(http.StatusCreated, "created")
 }
 
+func createNonCancelCtx() nonCancellableContext {
```

Comment:
It's typical to use `New_____()` for a "constructor" function names.

https://google.github.io/styleguide/go/decisions#package-vs-exported-symbol-name 
or
https://go.dev/doc/effective_go#composite_literals